packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/editModal.css

```diff
@@ -0,0 +1,73 @@
+.root {
+    background-color: white;
+    height: auto;
+    left: calc(50% - 35vw);
+    max-height: 85vh;
+    max-width: 70vw;
```

Comment:
The desktop modal should probably have a pixel max-width. I know an ultrawide isn't the best example of the average user but even a high resolution/large monitor will force the input fields to be extremely wide.

[![Image from Gyazo](https://i.gyazo.com/3f86f32fc5067e727ca73a5ab1bcf2c0.png)](https://gyazo.com/3f86f32fc5067e727ca73a5ab1bcf2c0)

Reply:
Changed to fixed width in 456f4e8.